src/rocker/cli.py

```diff
@@ -37,6 +39,16 @@ def main():
     parser.add_argument('--network', choices=['bridge', 'host', 'overlay', 'none'])
     parser.add_argument('--devices', nargs='*')
 
+    # Check for docker access
+    client = get_docker_client()
+    try:
+        client.containers()
```

Comment:
Recommend moving this check to after `parser.parse_args()` so `rocker --help` works without trying to connect to a docker daemon.